azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -24,7 +26,7 @@
 
 public class InteractiveTestJob extends AbstractProcessJob {
 
-  private static final ConcurrentHashMap<String, InteractiveTestJob> testJobs =
+  public static final ConcurrentHashMap<String, InteractiveTestJob> testJobs =
```

Comment:
Sorry for that. It was made public so that elsewhere one can do `synchronized (testJobs)` and `testJobs.wait()` to be notified immediately of an update. Of course better would be to have that method in `InteractiveTestJob` itself, so that `testJobs` could still be private.

Static: I guess that's just been the easy way for someone to implement this, but it definitely wouldn't have to be so.